de.metas.payment.esr/src/main/java/de/metas/payment/esr/api/impl/ESRBPBankAccountDAO.java

```diff
@@ -64,6 +69,57 @@
 				.list();
 	}
 
+	/**
+	 * Formatting done according to https://www.gkb.ch/de/Documents/DC/Beratung-Produkte/Factsheets-Flyers/Handbuch-ESR/ESR-Handbuch-Postfinance-DE.pdf
```

Comment:
```suggestion
	 * Explode the given {@code esrString} into a number of syntactically equivalent strings that can be matched against {@code C_BP_BankAccount.ESR_RenderedAccountNo}.
	 * Formatting done according to https://www.gkb.ch/de/Documents/DC/Beratung-Produkte/Factsheets-Flyers/Handbuch-ESR/ESR-Handbuch-Postfinance-DE.pdf
```

Reply:
added the explanation. now that i've read yours it makes so much sense what you wrote, but yesterday i totally couldn't figure out a good explanation.